packages/kaoto-ui/cypress/support/kaoto-ui-commands/default.js

```diff
@@ -1,5 +1,27 @@
 import 'cypress-file-upload';
 
+Cypress.Commands.add('isomorphicGet', (selector) => {
```

Comment:
@tplevko here we have the function that I created for searching the iframe. It leverages an environment variable `IFRAME_ENABLED` to look for the multiplying architecture's iframe or look directly over the root document.